freqtrade/configuration/configuration.py

```diff
@@ -465,6 +465,15 @@ def _process_analyze_options(self, config: Config) -> None:
         self._args_to_config(config, argname='timerange',
                              logstring='Filter trades by timerange: {}')
 
+        self._args_to_config(config, argname='analysis_rejected',
+                             logstring='Analyse rejected signals: {}')
+
+        self._args_to_config(config, argname='analysis_to_csv',
+                             logstring='Store analysis tables to CSV: {}')
+
+        self._args_to_config(config, argname='analysis_csv_path',
```

Comment:
I'm not sure i'd make this configurable.
it adds little benefit in my eyes (it still won't allow you to customize the filename) - so backtest_result is as good a name as any.

Reply:
OK so you suggest simply using user_data/backtest_result even if export-dir is supplied?

Reply:
well using export_dir is fine i guess? so the file is where all the other results are.

Currently - we're allowing someone to set "store this in cwd" (`--analysis_csv_path $(pwd)`) - but the filename will be static. 
So i have no way to prevent overwriting, unless i want a separate directory for every file.

if any, then a "csv path+name" argument would make more sense  - allowing me to customize the name (if i want to prevent the overwriting for specific results).